cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/ApacheKafkaTopicService.java

```diff
@@ -251,6 +249,9 @@ public synchronized ApiResponse deleteTopic(ClusterTopicRequest clusterTopicRequ
         Thread.currentThread().interrupt();
         errorMessage = e.getMessage();
       }
+      if ((e.getMessage().contains("UnknownTopicOrPartition"))) {
+        return ApiResponse.builder().success(true).message(ApiResultStatus.SUCCESS.value).build();
+      }
       log.error("Unable to delete topic {}, {}", clusterTopicRequest.getTopicName(), errorMessage);
```

Comment:
Does this handle where we delete a topic, that deletion is not synced with the Klaw DB and so this will then allow Klaw to remove it from the DB?

Reply:
Yes, it will respond with a success even when topic is not found. Alternatively, an admin can remove it from klaw from sync page.